GithubRepository/src/githubApi.js

```
@@ -1,80 +1,59 @@
-class GitHubClient {
-    constructor() {
-        this.token = process.env.GITHUB_TOKEN;;
-        this.baseUrl = 'https://api.github.com';
-    }
+const fs = require('fs');
 
-    async testConnection() {
-        if (!this.token) {
-            console.error('No se ha proporcionado un token de acceso. Configure la variable de entorno GITHUB_TOKEN.');
-            return false;
-        }
+const token = process.env.GITHUB_TOKEN;
+const baseUrl= 'https://api.github.com';
+const org_Name = 'stackbuilders';
+const baseUrlOrg = `https://api.github.com/orgs/${org_Name}/repos`;
 
-        try {
-            const response = await fetch(`${this.baseUrl}/user`, {
-                headers: {
-                    Authorization: `token ${this.token}`
-                }
-            });
+    async function testConnection(token) {
+    if (!token) {
+        console.error('No se ha proporcionado un token de acceso. Configure la variable de entorno GITHUB_TOKEN.');
+        return false;
+    }
 
-            if (response.ok) {
-                const userData = await response.json();
-                console.log('Conexión exitosa. Bienvenido, ' + userData.login);
-                return true;
-            } else {
-                console.error('Error de conexión:', response.statusText);
-                return false;
+    try {
+        const response = await fetch(`${baseUrl}/user`, {
+            headers: {
+                Authorization: `token ${token}`
             }
-        } catch (error) {
-            console.error('Error de conexión:', error.message);
+        });
+
+        if (response.ok) {
+            const userData = await response.json();
+            console.log('Conexión exitosa. Bienvenido, ' + userData.login);
+            return true;
+        } else {
+            console.error('Error de conexión:', response.statusText);
             return false;
```

better to throw an error here